datm/cime_config/config_component.xml

```diff
@@ -217,19 +218,20 @@
     <values match="last">
       <value compset="2000.*_DATM%1PT">$DATM_YR_START</value>
       <value compset="1850.*_DATM%QIA">1</value>
-      <value compset="1850.*_DATM%CRU">1</value>
+      <value compset="1850.*_DATM%CRUJRA2024">1</value>
+      <value compset="1850.*_DATM%CRUv7">1</value>
       <value compset="1850.*_DATM%GSW">1</value>
       <value compset="HIST.*_DATM%QIA">1895</value>
-      <value compset="HIST.*_DATM%CRU">1901</value>
+      <value compset="HIST.*_DATM%CRUJRA2024">1901</value>
+      <value compset="HIST.*_DATM%CRUv7">1901</value>
       <value compset="HIST.*_DATM%GSW">1901</value>
       <value compset="HIST.*_DATM%NLDAS2">$DATM_YR_START</value>
       <value compset="20TR.*_DATM%QIA">1895</value>
-      <value compset="20TR.*_DATM%CRU">1901</value>
+      <value compset="20TR.*_DATM%CRUJRA2024">1901</value>
+      <value compset="20TR.*_DATM%CRUv7">1901</value>
       <value compset="20TR.*_DATM%GSW">1901</value>
       <value compset="SSP.*_DATM%QIA">$DATM_YR_START</value>
-      <value compset="SSP.*_DATM%CRU">$DATM_YR_START</value>
       <value compset="SSP.*_DATM%GSW">$DATM_YR_START</value>
-      <value compset="1850.*_DATM%CRU">1</value>
       <value compset="1850.*_DATM%GSW">1</value>
```

Comment:
Noticing a couple of details:
- This line seems duplicated so delete this one.
- Above this line is a section of SSP compsets where I should probably add CRUJRA2024 and CRUv7.
- See if I missed anything else...